modules/trusted-firmware-m/Kconfig

```diff
@@ -207,4 +207,13 @@ config ROM_START_OFFSET
 
 endif # !TFM_BL2
 
+config TFM_FLASH_MERGED_BINARY
+	bool "Combine with the TF-M and BL2 binaries when flashing the application"
+	help
```

Comment:
Should we have `depends on TRUSTED_EXECUTION_NONSECURE` ?

Reply:
It's inside the `if BUILD_WITH_TFM` which has `depends on TRUSTED_EXECUTION_NONSECURE`